### PR TITLE
Support log_tag defaults from containers.conf

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://github.com/containers/podman/issues/10204

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1764284, please set that to POST and assign to Jindrich once this merges.

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Review by ashley-cui — COMMENTED** [review]
LGTM

File: pkg/specgen/generate/container.go
Comment (saschagrunert) [inline pkg/specgen/generate/container.go:216]: Since the field name is `log_driver`, just a nit:
```suggestion
			logrus.Warnf("log_tag %q is not allowed with %q log_driver", rtc.Containers.LogTag, define.JSONLogging)
```

**Review by saschagrunert — APPROVED** [review]
A nit, otherwise LGTM

**Pull request merged**
